content/chainguard/chainguard-images/network-requirements.md

```diff
@@ -48,9 +48,11 @@ This table lists the third-party DNS hostnames, associated ports, and protocols
 
 | Hostname                                                  | Port | Protocol | Notes                        |
 | --------------------------------------------------------- | ---- | -------- | ---------------------------- |
-| 9236a389bd48b984df91adc1bc924620.r2.cloudflarestorage.com | 443  | HTTPS    | Blob storage for cgr.dev     |
+| 9236a389bd48b984df91adc1bc924620.r2.cloudflarestorage.com | 443  | HTTPS    | Blob storage for *.cgr.dev   |
 | support.chainguard.dev                                    | 443  | HTTPS    | Support access for customers |
 
+> Note that the `.r2.cloudflarestorage.com` host is used to serve both image data and packages via `*.cgr.dev`.
```

Comment:
Is there a reason you didn't include the 9236... here? It might be interpreted as a typo of '*' being dropped.